Scripts/AwsEC2GetPublicSGRules/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## [Unreleased]
```

Comment:
please remove this file.
CHANGELOG files are created automatically for new content entities during content release.
those files should only be modified upon updates and not creation.

Reply:
Got it. Updated the PR with changelog removed